frontend/src/data/nsf23313_2_1/categories/values.ts

```diff
@@ -0,0 +1,29 @@
+
+export const Categories = {
+    'gender': new Set<string>([
+        'Male',
+        'Female',
+    ]),
+    'race': new Set<string>([
+        'NativeHawaiianOrOtherPacificIslanderAlone',
+        'Black',
+        'TwoOrMoreRaces',
+        'WhiteAlone',
+        'Asian',
+        'AmericanIndianOrAlaskaNativeAlone',
+        'HispanicOrLatino',
+        'UnknownRace',
+    ]),
+    'education': new Set<string>([
+        'DoctorateDegree',
+        'MastersDegree',
+    ]),
+    'citizenship': new Set<string>([
+        'Citizen',
+        'VisaHolder',
+    ]),
+    'occupation': new Set<string>([
```

Comment:
I think these are each their own category. See [here](file:///home/drumsound/Downloads/nsf22319-tab002-001.pdf) for details. The sum under each column add to 100%

Reply:
separated the categories